utils/scripts/markdown-table.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+grep \
+  --exclude-dir=.template \
+  --exclude-dir=node_modules \
+  --include=\*.js \
+  --exclude=\*.spec.js \
+  -rnw 'packages' \
+  -e 'const COMPONENT_ID = ' \
+  | rev | cut -d: -f1 | rev \
+  | sed 's/const COMPONENT_ID = //g' | sed 's/;/,/g' | sort | tr -d '\n' \
+  | rev |  cut -c 2- | rev \
+  | awk ' BEGIN {print "["}{print} END {print"]"}' | sed "s/'/\"/g" \
+  | node ./utils/scripts/generate-markdown-table.js
```

Comment:
Please suggest improvements this is a little convoluted basically it scans all js files for `const COMPONENT_ID = ` and extracts the names and returns a string in js array format e.g.

```js
["avatars.avatar", "buttons.button"]
```

Reply:
Improved in d9baaee

Reply:
Adding a comment would go a long way – i.e. "find and format all COMPONENT_ID constants as an array to be consumed by..."